src-built-in/components/advancedAppCatalog/src/components/Showcase/Header.jsx

```diff
@@ -69,7 +69,9 @@ const Header = props => {
 						)}
 				</button>
 				{props.installed && <button className={props.entitled ? "action-button remove" : "action-button remove disabled"} disabled={!props.entitled} onClick={removeApp}>
-					<div className='ff-adp-trash-outline' title='Remove App'/>
+					<div title='Remove App'>
```

Comment:
Pre-existing issue, this isn't something to solve here. However this is bad accessibility.

1) This should just be a `<button>`
2) Title shouldn't be necessary when the textContent is clear or the element specifically kinda needs it for the a11y tree (`<nav>` for example.)

Reply:
It is a button